examples/domainmodel/syntaxes/domain-model.tmLanguage.json

```diff
@@ -13,36 +17,5 @@
       "name": "keyword.symbol.domain-model",
       "match": "(\\:|\\.|\\{|\\})"
     }
-  ],
-  "repository": {
-    "comments": {
-      "patterns": [
-        {
-          "name": "comment.block.domain-model",
-          "begin": "/\\*",
-          "beginCaptures": {
-            "0": {
-              "name": "punctuation.definition.comment.domain-model"
-            }
-          },
-          "end": "\\*/",
-          "endCaptures": {
-            "0": {
-              "name": "punctuation.definition.comment.domain-model"
-            }
-          }
-        },
-        {
-          "begin": "(^\\s+)?(?=//)",
-          "beginCaptures": {
-            "1": {
-              "name": "punctuation.whitespace.comment.leading.cs"
-            }
-          },
-          "end": "(?=$)",
-          "name": "comment.line.domain-model"
-        }
-      ]
-    }
-  }
+  ]
```

Comment:
Multiline comment highlighting does not work with these changes.
![Screen Shot 2021-09-28 at 15 09 44](https://user-images.githubusercontent.com/4067210/135093259-14344fa1-4edf-4849-853b-28480acbf1e3.png)

Reply:
True, the textmate implementation only looks at a single line at a time. For multiline comments we need to use the repository. That requires some extensive changes in order to calculate how a comment starts and ends. I implemented some heuristic for that.